### PR TITLE
Prepare for WP 5.6

### PR DESCRIPTION
**Pull request by peterwilsoncc**
* Confirm the new auto-updates interface is not shown
* Remove site health checks relating to background updates
* Block all users from the following caps:
  * update_plugins
  * delete_plugins
  * install_plugins
  * upload_plugins
  * update_themes
  * delete_themes
  * install_themes
  * upload_themes
  * update_languages (cc @thomasgriffin should this include languages?)
  * install_languages
  * update_core
  * update_php

### CODE REVIEW

File: disable-all-wp-updates.php [hunk 1]
```diff
@@ -182,6 +189,76 @@ public function admin_init() {
...
 
+	/**
+	 * Block users from taking certain actions.
+	 *
+	 * Adds `do_not_allow` magic keyword to block users from taking update
+	 * related actions.
+	 *
+	 * @since 1.1.0
+	 *
+	 * @param string[] $caps Required primitive caps for the capability been checked.
```
Comment (thomasgriffin) [inline disable-all-wp-updates.php:200]: Is `string[]` correct? Is it not an `array` being passed?

File: disable-all-wp-updates.php [hunk 2]
```diff
@@ -182,6 +189,76 @@ public function admin_init() {
...
+	 * @param string   $cap  The capability been checked.
+	 * @return string[] Modified array of primitive caps.
```
Comment (thomasgriffin) [inline disable-all-wp-updates.php:202]: Same thing as above.

File: disable-all-wp-updates.php (hunk 1 above)
Comment (peterwilsoncc) [reply disable-all-wp-updates.php:200]: The `[]` indicates array of type `string`. I think it's a relatively recent addition to PHP docblock standards so IDE's can catch errors. https://docs.phpdoc.org/3.0/guide/guides/types.html#arrays

**Review by thomasgriffin — APPROVED** [review]

**Comment by thomasgriffin** [comment]
Approved the changes. If this will be the only PR in the next release, go ahead and include readme updates in it as well.

**Comment by peterwilsoncc** [comment]
Thanks for the prompt review: I've made the readme.txt changes in https://github.com/thomasgriffin/disable-all-wp-updates/pull/4/commits/bbf44c8a216200464a0eefb9e38b5447802ffdd9

**Pull request merged**